docs/eng-how-to.md

```diff
@@ -0,0 +1,167 @@
+# Overview
+
+The Science Design System (SDS) brings consistency and universal standards to CZI’s science products by offering a library of high quality, reusable components that deliver predictable, accessible and easy to learn experiences. Our goal is to democratize access to tools and technologies for scientists.
+
+# Getting Started
+
+## Installation
+
+NPM Package
+
+```
+// with npm
+npm i czifui
+
+// with yarn
+yarn add czifui
+```
+
+**Currently SDS uses Material UI v4**
+
+czifui installs without peer dependencies to prevent version errors. Please ensure the following dependencies are also installed:
```

Comment:
```suggestion
czifui installs without direct dependencies to prevent version errors. Please ensure the following peer dependencies are also installed:
```